shared/chat/conversation/messages/coinflip/results.js

```diff
@@ -159,27 +160,41 @@ const CoinFlipResultCoin = (props: CoinType) => (
 type HandType = {|
   hands: ?Array<RPCChatTypes.UICoinFlipHand>,
 |}
-const CoinFlipResultHands = (props: HandType) => (
-  <Kb.Box2 direction="horizontal" fullWidth={true}>
-    <Kb.Box2 direction="vertical" gap="tiny">
-      {props.hands &&
-        props.hands.map(hand => (
-          <Kb.Box2 direction={Styles.isMobile ? 'vertical' : 'horizontal'} fullWidth={true} key={hand.target}>
-            <Kb.Box2
-              alignItems="flex-start"
-              direction={Styles.isMobile ? 'vertical' : 'horizontal'}
-              style={styles.handTarget}
-            >
-              <Kb.Text selectable={true} type="BodyBig">
-                {hand.target}
-              </Kb.Text>
-            </Kb.Box2>
-            <CoinFlipResultDeck deck={hand.hand} hand={true} />
-          </Kb.Box2>
-        ))}
+const CoinFlipResultHands = (props: HandType) => {
+  if (!props.hands) return null
+  const handsWithCards = props.hands.filter(hand => hand.hand)
+  const handsWithoutCards = props.hands ? props.hands.slice(handsWithCards.length) : []
```

Comment:
why not just filter the opposite (or use partition from lodash)

Reply:
Good point.